ideascube/serveradmin/catalog.py

```diff
@@ -427,26 +440,41 @@ def _progress(*args):
 
     def list_installed(self, ids):
         pkgs = self._get_packages(
-            ids, self._catalog['installed'], fail_if_no_match=False)
+            ids, self._catalog['installed'],
+            fail_if_no_match=False, fail_if_invalid_package=False)
         return sorted(pkgs, key=attrgetter('id'))
 
     def list_available(self, ids):
         pkgs = self._get_packages(
-            ids, self._catalog['available'], fail_if_no_match=False)
+            ids, self._catalog['available'],
+            fail_if_no_match=False, fail_if_invalid_package=False)
         return sorted(pkgs, key=attrgetter('id'))
 
     def list_upgradable(self, ids):
         pkgs = []
 
         for ipkg in self._get_packages(
-                ids, self._catalog['installed'], fail_if_no_match=False):
+                ids, self._catalog['installed'],
+                fail_if_no_match=False, fail_if_invalid_package=False):
             upkg = self._get_package(ipkg.id, self._catalog['available'])
 
             if ipkg != upkg:
                 pkgs.append(upkg)
 
         return sorted(pkgs, key=attrgetter('id'))
 
+    def list_nothandled(self, ids):
```

Comment:
I'm also not fan of silently ignoring content.

- This can be useful for debug/analysis ("hello devs, I can't install content on my box, and I know that this content is existing")
- This is a clue that something is not up to date.